docs/metrics/prometheus/grafana/README.md

```diff
@@ -14,3 +14,10 @@ Visualize MinIO metrics with our official Grafana dashboard available on the [Gr
 Refer to the dashboard [json file here](https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/minio-overview.json).
 
 ![Grafana](https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/grafana-minio.png)
+
+
+Replication metrics can also be viewed in grafana dashboard using [json file here](https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/minio-replication.json).
+
+
+
+![Grafana](https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/grafana-replication.png)
```

Comment:
The screenshot doesn't show any replication stuff - are you sure this is the correct screenshot?

Reply:
The screen shot has `Sent Bytes for Replication` and `Received Bytes for Replication` . The screenshot is large , You need to scroll. 

![Screenshot from 2021-11-17 23-08-09](https://user-images.githubusercontent.com/13465383/142253740-acac45c0-de18-4ae1-894f-b5918fb8a2df.png)

![Screenshot from 2021-11-17 23-08-17](https://user-images.githubusercontent.com/13465383/142253756-1e8f9e3c-6933-4375-989c-d9664f35de8e.png)